### DISCUSSION TITLE
增加更多的单点登录，例如Casdoor

### DISCUSSION THREAD

**Discussion by chung1912** [lobehub/lobehub#3846] [Self Hosting] [answered]
目前本地化部署原生只支持logto，但这个是真不太好用，其他的都是联网认证。
建议增加casdoor认证，最好能支持它的webhook

**Comment by arvinxx** [comment, marked as answer]
新版文档已上线，推荐使用 Casdoor。 https://lobehub.com/zh/docs/self-hosting/server-database/docker-compose
